vaadin-components-gwt/src/main/webapp/vaadin-grid/vaadin-grid.html

```diff
@@ -490,6 +490,56 @@
     min-width: 30px;
   }
 
+
+  .v-grid-sidebar {
+    z-index: 5;
+    position: absolute;
+    top: 0;
+    right: 0;
+  }
+
+  .v-grid-sidebar-button {
+    width: 56px;
+    height: 56px;
+    border: none;
+    background: none;
+    outline: none;
+  }
+
+  .opened .v-grid-sidebar-button {
+    background-color: #eeeeee;
+  }
+
+  .v-grid-sidebar-button:after {
+    content: url(ic_view_column_black_24px.svg);
+  }
+
+  .v-grid-sidebar-content {
+    background-color: #eeeeee;
+    width: 200px;
```

Comment:
Could be min-width, so that longer column titles still fit without clipping/ellipsis.